tools/src/main/java/org/apache/kafka/tools/VerifiableConsumer.java

```diff
@@ -614,6 +616,14 @@ private static ArgumentParser argParser() {
                 .type(String.class)
                 .metavar("CONFIG_FILE")
                 .help("Consumer config properties file (config options shared with command line parameters will be overridden).");
+        
+        parser.addArgument("--metadata-recovery-strategy")
```

Comment:
Could you please consider using a configuration file to pass the rebootstrap settings? Since this configuration is not commonly used, it may not be worthwhile to expose it as an individual option.